cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -147,7 +147,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		for i, languageID := range command.Languages {
 			assessmentLanguages[i] = language.Languages[languageID]
 		}
-		assessments = report.NewAssessmentPerModelPerLanguagePerRepository(assessmentModels, assessmentLanguages, append(command.Repositories, repositoryPlainName))
```

Comment:
Changed this quite a bit. Also in https://github.com/symflower/eval-dev-quality/pull/53/commits/c2d3dcab9c7a56eb82910e06ca79f8f10af7b5fe